tests/integration_tests/functional/test_net.py

```diff
@@ -43,7 +43,7 @@ def test_high_ingress_traffic(uvm_plain_any):
     # Start iperf3 client on the host. Send 1Gbps UDP traffic.
     # If the net device breaks, iperf will freeze. We have to use a timeout.
     utils.check_output(
-        "timeout 31 {} {} -c {} -u -V -b 1000000000 -t 30".format(
+        "{} {} -c {} -u -V -b 1000000000 -t 30".format(
```

Comment:
Need to remove the comment as well.

Reply:
fascinating how I completely manage to fail to read that comment when writing the commit :joy: 

oh well, pytest timeout will do the job better anyway lol